compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -195,12 +195,26 @@ fn make_format_args(
                                     Applicability::MaybeIncorrect,
                                 );
                             } else {
-                                let sugg_fmt = match args.explicit_args().len() {
-                                    0 => "{}".to_string(),
-                                    count => {
-                                        format!("{}{{}}", "{} ".repeat(count))
+                                let is_empty_tuple = |kind: &ExprKind| -> bool {
+                                    match kind {
+                                        ExprKind::Block(b, _) if b.stmts.is_empty() => true,
```

Comment:
Question: will this fire on `'label: {}`? Maybe we want `ExprKind::Block(b, None)` here?

Reply:
> Question: will this fire on `'label: {}`? Maybe we want `ExprKind::Block(b, None)` here?

Done. Thank you.